### PR TITLE
Coworking Jan 2023 updates from NMFS fall cohorts

### PR DESCRIPTION
**Pull request by jules32**
First commit is agenda 3

### CODE REVIEW

**Comment by jules32** [comment]
Tomorrow Julie will make a typo and Stef will make the change as a commit during the PR review process 🎉 
- making a comment "please explain this, add a link to this"
- when she fixes the typo, she'll do that herself, make a suggested commit. and when submit the review Julie will be notified and can accept the commit rather than Julie having to fix herself

File: README.md [hunk 1]
```diff
@@ -212,3 +214,25 @@ Then, to move this to a Google Doc and fine-tune formatting, follow these notes
 **How to "update heading 1 to match"**: (Stef notes this has not been working in these docs) In Google Doc, to update a text style (headings, normal text, with font type, size etc), highlight a section with the style you want, click the styles dropdown shown in the screenshot, and select e.g. "Update Heading 1 to match". Double check the doc because we've noticed it missed some in an agenda.
 
 ![Screen Shot 2022-07-14 at 5 50 28 PM](https://user-images.githubusercontent.com/11927811/179125336-ec2fc1e5-792e-495d-8a29-c1ed3ec3cdc4.png)
+
+## R package developer notes
+
+Steps to edit `kbyer` locally and review those edits:
```
Comment (stefaniebutland) [inline README.md:220]:
```suggestion
Steps to edit `kyber` locally and review those edits: 
```

File: README.md [hunk 2]
```diff
@@ -212,3 +214,25 @@ Then, to move this to a Google Doc and fine-tune formatting, follow these notes
 **How to "update heading 1 to match"**: (Stef notes this has not been working in these docs) In Google Doc, to update a text style (headings, normal text, with font type, size etc), highlight a section with the style you want, click the styles dropdown shown in the screenshot, and select e.g. "Update Heading 1 to match". Double check the doc because we've noticed it missed some in an agenda.
 
 ![Screen Shot 2022-07-14 at 5 50 28 PM](https://user-images.githubusercontent.com/11927811/179125336-ec2fc1e5-792e-495d-8a29-c1ed3ec3cdc4.png)
+
+## R package developer notes
+
+Steps to edit `kbyer` locally and review those edits: 
+
+1. In RStudio, open `kyber` project, pull main branch
```
Comment (stefaniebutland) [inline README.md:222]: This bulleted list is spectacular

**Review by stefaniebutland — CHANGES_REQUESTED** [review]

**Comment by jules32** [comment]
FYI @seankross - 
Hi! I've updated some text (not code) here following our fall cohorts :)

**Pull request merged**